mifospay/src/main/java/org/mifos/mobilewallet/mifospay/standinginstruction/ui/NewSIActivity.kt

```diff
@@ -35,13 +45,21 @@ class NewSIActivity : BaseActivity(), StandingInstructionContract.NewSIView {
         setToolbarTitle(getString(R.string.tile_si_activity))
         showColoredBackButton(Constants.BLACK_BACK_BUTTON)
         mPresenter.attachView(this)
+        initView()
     }
 
     override fun setPresenter(presenter: StandingInstructionContract.NewSIPresenter) {
         this.mNewSIPresenter = presenter
     }
 
-    @OnClick(R.id.btn_valid_till)
+    private fun initView(){
```

Comment:
There are some code styling issues here. I am nitpicking [here](https://checkstyle.sourceforge.io/styleguides/google-java-style-20180523/javaguide.html#s4.6.2-horizontal-whitespace) but do see this site, it will be a good read.